src/components/Textarea.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+export default function Textarea({ limit }) {
+  const [textareaValue, setTextareaValue] = useState('');
+
+  const textareaId = `textarea-${document.querySelectorAll('textarea').length}`;
```

Comment:
Won't it be the same id for all textareas?

e.g.

```html
<textarea/>
<textarea/>
<textarea/>
```

document.querySelectorAll(`textarea`).length === 3

or am I wrong?

wow. look what I found. Let's try [this](https://react.dev/reference/react/useId)

Reply:
Yes, there is a bug. I didn't know about useId, I'll think about it